src/modules/taskList.js

```diff
@@ -0,0 +1,53 @@
+import Task from './task.js';
+
+class TaskList {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  saveTasks() {
+    localStorage.setItem('tasks', JSON.stringify(this.tasks));
+  }
+
+  addTask(description) {
+    const newTask = new Task(description);
+    this.tasks.push(newTask);
+    this.saveTasks();
+  }
+
+  deleteTask(index) {
+    this.tasks.splice(index, 1);
+    this.updateIndexes();
+    this.saveTasks();
+  }
+
```

Comment:
- [ ] You have done a fantastic job on this milestone. However, it is recommended in the project's requirements that all the properties should appear including the `index` after adding a task. However, in your case, the index property only appears after deleting a task. Kindly rectify that.

### After adding tasks

![image](https://github.com/Abbas-Aljuaifari/To-Do-List/assets/98453979/cc82e335-b9a8-4749-88a4-61b794d121f6)

### After Deleting one task

![image](https://github.com/Abbas-Aljuaifari/To-Do-List/assets/98453979/930620f7-a6b4-4d8e-b19a-388d16585861)